polymath/polynomial.py

```
@@ -304,22 +386,28 @@ def __mul__(self, arg):
-                                unit=Unit._mul_units(self._unit, arg._unit))
+            nself = self._values.shape[-self._drank - 1]
+            narg = arg._values.shape[-self._drank - 1]
+
+            # Standard convolution: result[k] = sum of self[i] * arg[j] where i+j = k
+            # But in decreasing order, position 0 is highest power
+            # So if self has coefficients [a_1, a_0] and arg has [b_1, b_0],
+            # result[0] = a_1*b_1, result[1] = a_1*b_0 + a_0*b_1, result[2] = a_0*b_0
+            for i in range(nself):
+                for j in range(narg):
+                    k = i + j
+                    self_indx = (Ellipsis, i) + tail_indx
```

Ruff (0.14.7)

Consider `(Ellipsis, i, *tail_indx)` instead of concatenation

Replace with `(Ellipsis, i, *tail_indx)`

(RUF005)